.github/workflows/integration-test-workflow.yml

```diff
@@ -31,6 +31,16 @@ jobs:
           name: debian-packages-amd64
           path: debian-package_unpack
 
+      - name: disconnect c8y
+        run: sudo tedge disconnect c8y
+        # We need to continue when there is no tedge already installed
+        continue-on-error: true
```

Comment:
if continued on error, don't we get into the same issue again? if `tedge disconnect` fails.

Reply:
Yes, if disconnect fails we get the issue again. That's why we should ensure somewhere else that it works. We can only continue on error here as it could be the case that tedge is not installed (at least with a linear workflow on GH)